keras_hub/src/models/rwkv7/rwkv7_tokenizer_test.py

```diff
@@ -0,0 +1,59 @@
+from keras_hub.src.models.rwkv7.rwkv7_tokenizer import RWKVTokenizer
+from keras_hub.src.tests.test_case import TestCase
+
+
+class RWKVTokenizerTest(TestCase):
+    def setUp(self):
+        self.vocab = [
+            "0 '\\n' 1",
+            "1 ' ' 1",
+            "2 'the' 3",
+            "3 'hello' 5",
+            "4 'world' 5",
+        ]
+        self.tokenizer = RWKVTokenizer(vocabulary=self.vocab)
```

Comment:
This is veru basic vocab for test, include the changes like we have for other tests [Test](https://github.com/keras-team/keras-hub/blob/master/keras_hub/src/models/llama3/llama3_tokenizer_test.py)